lib/ActivityStreams.js

```diff
@@ -113,6 +114,12 @@ function ActivityStreams(metadataStore, options = {}) {
   this._setupButton();
   NewTabURL.override(this._newTabURL);
   this._setHomePage();
+  this.prefsProvider = new PrefsProvider({
```

Comment:
question/nit Any reason why this isn't following the same underscore naming convention like the other instances initiated in the constructor?